### PR TITLE
CSP-117 get providers from roatpservice code and coverage

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/RegisteredProviderModel.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace SFA.DAS.Roatp.CourseManagement.InnerApi.Models.RegisteredProvider
+{
+    public class RegisteredProviderModel
+    {
+        public int ukprn { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/RegisteredProviderModel.cs:7]: Case

File: src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/Feedback.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.Roatp.CourseManagement.InnerApi.Models.RegisteredProvider
+{
+    public class Feedback
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/Feedback.cs:5]: Think we should use proper name EmployerFeedback

**Review by DevenKShah — APPROVED** [review]

File: src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/Feedback.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/Feedback.cs:5]: I have used the names that come back from roatp-service exactly, so diverging will cause us mapping and mapping back issues if we diverge, I suspect.  I'll leave it as is.

File: src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/RegisteredProviderModel.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/RegisteredProviderModel.cs:7]: yep

**Review by DevenKShah — APPROVED** [review]

**Comment by MarkFCain** [comment]
merge update issues, deleting to create a fresh version

**Pull request closed without merging**